### DISCUSSION TITLE
Loading

### DISCUSSION THREAD

**Discussion by Marsin77** [dewanakl/undangan#31] [General]
Lama loading 
![Uploading Screenshot_2024-07-10-12-29-16-47_40deb401b9ffe8e1df2f1cc5ba480b12.jpg…]()

**Comment by dewanakl** [comment]
Maaf, untuk screenshot nya belum ke upload dengan benar

Pastikan juga lokasi deployment ada pada singapore

**Comment by Marsin77** [comment]
![Screenshot_2024-07-10-12-29-16-47_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/dewanakl/undangan/assets/175084216/38b4fda7-1d6c-497c-b003-6e9163e9ebac)

**Reply by dewanakl** [reply]
Bisa inspect element kemudian check image mana yang tidak bisa

**Comment by Marsin77** [comment]
Sudah betul mi kak
![Uploading Screenshot_2024-07-10-14-22-15-36_40deb401b9ffe8e1df2f1cc5ba480b12.jpg…]()

**Comment by Marsin77** [comment]
![Screenshot_2024-07-10-14-22-15-36_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/dewanakl/undangan/assets/175084216/fa137bce-8ec7-468c-8c7b-9df229468ad7)

**Reply by dewanakl** [reply]
untuk src image lebih baik di tulis secara lengkap saja kak

**Comment by Marsin77** [comment]
Kaka, kenapa lodding terus
![Screenshot_2024-12-20-20-25-49-88_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/56180054-6d2d-442f-a470-d10f29e76890)
